src/index.html

```diff
@@ -1,2 +1,18 @@
+<HTML>
+	<HEAD>
+		<script type = "text/javascript" src="index.js"></script>
+	</HEAD>
+	<BODY>
+
 <h1>Semantic HTML tags</h1>
 <h2>How many do you know?</h2>
+<form>
+	<input type="text" id="input1">
+	<button type="submit" onclick = "submitVal()" >Submit</button>
```

Comment:
Stay consistent by removing unnecessary spaces like in this click function, as well as in the script (please check your code in general for this one).